src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_acl.erl

```diff
@@ -254,7 +254,12 @@ names_to_ids(ACE, OrgId) ->
             safe_fetch_ids(user, undefined, ej:get({<<"users">>}, ACE)) ++
             safe_fetch_ids(client, OrgId, Clients)
     end,
-    GroupIds = safe_fetch_ids(group, OrgId, ej:get({<<"groups">>}, ACE)),
+    %% Global groups can be intermixed with org local groups. This creates some ambiguity as
```

Comment:
> It's obscure and not likely to be hit by many people

Let's do a search against hosted data to see if we have any conflicting group names.

Also, until we have a solution that allows us to distinguish global vs local groups in user-provided acl input, maybe we should disallow people from creating groups or actors that match various reserved names in the system?